h2o-core/src/main/java/hex/ModelMetricsBinomial.java

```diff
@@ -79,7 +80,21 @@ public ConfusionMatrix cm(AUC2.ThresholdCriterion criterion) {
   public double aucpr() { return auc_obj()._pr_auc; } // for compatibility with naming in ScoreKeeper.StoppingMetric annotation
   public double lift_top_group() { return gainsLift().response_rates[0] / gainsLift().avg_response_rate; }
 
-  /**
+  @Override
+  public boolean isEqualUpToTolerance(ComparisonUtils.MetricComparator comparator, ModelMetrics other) {
+    super.isEqualUpToTolerance(comparator, other);
+    ModelMetricsBinomial specificOther = (ModelMetricsBinomial)other;
+
+    comparator.compareValuesUpToTolerance("auc", this.auc(), specificOther.auc());
+    comparator.compareValuesUpToTolerance("pr_auc", this.pr_auc(), specificOther.pr_auc());
+    comparator.compareValuesUpToTolerance("lift_top_group", this.lift_top_group(), specificOther.lift_top_group());
+    comparator.compareValuesUpToTolerance("logloss", this.logloss(), specificOther.logloss());
+    comparator.compareValuesUpToTolerance("mean_per_class_error", this.mean_per_class_error(), specificOther.mean_per_class_error());
+    // CM, Gainslift, auc_obj are not supported yet.
```

Comment:
with the new interface, we can publish CM and others to the comparator and just ignore it there for now (this way anyone can provide the implementation without changes here - eg. SW can provide it)

Reply:
Ok, will change, thanks!